Dockerfile-backup-driver

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:focal
+FROM gcr.io/distroless/base:latest
```

Comment:
Please don't use any image with `latest` since it will constantly be re-tagged. You can consider the one velero image uses, https://github.com/vmware-tanzu/velero/blob/94549abd9097677b68ee0dac77fae8e08077ddd0/Dockerfile#L53

Reply:
Change to use base-debian10:nonroot.